build.zig

```diff
@@ -134,6 +134,8 @@ pub fn build(b: *std.build.Builder) void {
         lsm_forest_fuzz.setMainPkgPath("src");
         lsm_forest_fuzz.setTarget(target);
         lsm_forest_fuzz.setBuildMode(mode);
+        // Ensure that we get stack traces even in release builds.
+        lsm_forest_fuzz.omit_frame_pointer = false;
```

Comment:
The stacktraces are often printed wrong btw because of a stdlib bug, but they show up correctly in gdb.

Reply:
Good to know. And still better than no stack trace at all. This will be a big improvement for the VOPR, since it takes so much longer to run in debug mode than release.

Reply:
FWIW they should show up correctly in gdb even without `-fno-omit-frame-pointer`. If you configure your system to generate coredumps in a useful way it can certainly save a lot of time by examing the coredump of a crashed release VOPR run rather than waiting for a debug one to complete.